### ISSUE TITLE
BMP388 support

### ISSUE THREAD

**Issue by DaddyLlama** [pjalocha/esp32-ogn-tracker#52]
Hi,
Thank you for your work here. 
Will this support BMP388 sensors?

If not, How hard would it be to add BMP388 functionality? Is it as simple as listing it next to all entries where the BMP280 resides and add the Adafruit_BMP3x.cpp and Adafruit_BMP3x.h files in the sources?

**Comment by lupus78** [comment]
Yes, that's it. I did the MS5611 sensor code, this is my initial commit: https://github.com/pjalocha/esp32-ogn-tracker/commit/a0ba7ad4a3489c64dab836cdf6349a09be32ad92

just make and do a pull request. @pjalocha usually accept PRs
